modules/appeals_api/app/swagger/appeals_api/v1/higher_level_reviews_controller_swagger.rb

```diff
@@ -3,6 +3,8 @@
 class AppealsApi::V1::HigherLevelReviewsControllerSwagger
   include Swagger::Blocks
 
+  env_deprecation_state = Rails.env == 'dev' ? true : false
```

Comment:
can we change this to `!Settings.modules_appeals_api.documentation.path_enabled?`

Reply:
Unless you have a better way of removing the hard coded assumption about which environment we're deprecating in.

Reply:
path enabled returns a true or false, which we're using to turn v2 on in staging/dev/sandbox. 

If we're enabling v2 (true return value), then we're also deprecating v1 paths (false return value) - so we'd just invert the flag we're already using to determine the enabling of the documentation.

Reply:
I don't know if @nathanbwright was referring specifically to dev rather than `lower` environments. Have to ask him

Reply:
Perhaps we need to decide as a team how we want to roll this out.

Reply:
Not related to the discussion, but - reminder that the Rails environment is always `production` on all our servers. `Settings.vsp_environment` is used to determine the server environment.